File: weak-2/module-6/C_Even_Odd_Positive_and_Negative.py
t = int(input())
even = odd = positive = negative = 0

for i in range(t):
    n = int(input())
    if n % 2 == 0:
        even += 1
        if n < 0:
            negative += 1
        elif n > 0:
            positive += 1
    else:
        odd += 1
        if n < 0:
            negative += 1
        elif n > 0:
            positive += 1

print("Even:", even)
print("Odd:", odd)
print("Positive:", positive)
print("Negative:", negative)

File: Introduction To C/weak-2/module-6/C_Even_Odd_Positive_and_Negative.c
#include <stdio.h>
int main()
{
    int t, n, even = 0, odd = 0, positive = 0, negative = 0;
    scanf("%d", &t);
    for (int i = 1; i <= t; i++)
    {
        scanf("%d", &n);
        if (n % 2 == 0)
        {
            even++;
            if (n < 0)
            {
                negative++;
            }
            else if (n > 0)
            {
                positive++;
            }
        }
        else if (n % 2 != 0)
        {
            odd++;
            if (n < 0)
            {
                negative++;
            }
            else if (n > 0)
            {
                positive++;
            }
        }
    }
    printf("Even: %d\nOdd: %d\nPositive: %d\nNegative: %d\n", even, odd, positive, negative);
    return 0;
}